R/drive_publish.R

```diff
@@ -158,18 +161,26 @@ is_published_one <- function(id, name, verbose = TRUE) {
 
   if (verbose) {
     message_glue(
-      "The latest revision of file {sq(name)} is ",
+      "The latest revision of file {sq(file$name)} is ",
       "{if (proc_res$published) '' else 'NOT '}published."
     )
   }
-  publish_tbl(proc_res)
+  add_publish_cols(file, proc_res)
 }
 
-publish_tbl <- function(x) {
-  tibble::tibble(
-    check_time = Sys.time(),
-    revision = x$id,
-    published = x$published,
-    auto_publish = x$publishAuto %||% FALSE
+add_publish_cols <- function(d, x) {
+  if ("published" %in% names(d)) {
+    d$published_check_time <- Sys.time()
+    d$revision_id <- x$id
+    d$published <- x$published
+    d$auto_publish <- x$publishAuto %||% FALSE
+    return(d)
```

Comment:
I would order these by human interest. So `published` should come first, for example!